.evergreen/generate_evergreen_tasks.js

```diff
@@ -1,5 +1,6 @@
 const fs = require('fs');
```

Comment:
I did accidentally format and commit this file.  If this proves to be a pain to review, I'm happy to revert it.